torchbenchmark/models/pytorch_unet/__init__.py

```diff
@@ -89,6 +88,7 @@ def jit_callback(self):
             self.model = torch.jit.script(self.model)
 
     def eval(self) -> Tuple[torch.Tensor]:
+        torch.backends.cudnn.deterministic = True
```

Comment:
What about we remove line 91 ` torch.backends.cudnn.deterministic = True` AND line 9-10 `torch.backends.cudnn.benchmark = False` and `torch.backends.cudnn.deterministic = True` altogether?